downstream/assemblies/platform/assembly-ug-controller-job-templates.adoc

```diff
@@ -0,0 +1,55 @@
+[id="controller-job-templates"]
+
+= Job templates
+
+A job template is a definition and set of parameters for running an Ansible job. 
+Job templates are useful to execute the same job many times. 
+They also encourage the reuse of Ansible playbook content and collaboration between teams.
+
+The *Templates* list view shows job templates that are currently available. 
+The default view is collapsed (Compact), showing the template name, template type, and the timestamp of the last job that ran using that template. 
+You can click the arrow next to each entry to expand and view more information. 
+This list is sorted alphabetically by name, but you can sort by other criteria, or search by various fields and attributes of a template.
+
+image::ug-job-templates-home.png[Job templates home]
+
+From this screen, you can launch, edit, and copy a job template. 
+
+[NOTE]
+====
+Job templates can be used to build a workflow template.
+Templates that show the *Workflow Visualizer* icon next to them are workflow templates.
+Clicking the icon allows you to build a workflow graphically.
+Many parameters in a job template enable you to enable *Prompt on Launch* that can be modified at the workflow level, and do not affect the values assigned at the job template level. 
+For instructions, see the ADD XREF WHEN PAGE BUILT [Workflow Visualizer] section.
```

Comment:
Workflow vizualizer is in the Navigation panel, but the contents are in the Admin Guide (for now) it might be a while before an xref is available.